src/commands/stepfunctions/instrument.ts

```diff
@@ -155,6 +155,10 @@ export class InstrumentStepFunctionsCommand extends Command {
         stepFunctionTagsToAdd.push({key: TAG_VERSION_NAME, value: `v${cliVersion}`})
       }
 
+      if (!listStepFunctionTagsResponse?.tags?.some((tag) => tag.key === DD_TRACE_ENABLED)) {
```

Comment:
So, here, it can cover all the below three cases?
1) DD_TRACE_ENABLED not appear, then add tag true
2) DD_TRACE_ENABLED is set to true by customer, then add tag true
3) DD_TRACE_ENABLED is set to false by customer

Reply:
Thanks for catching this. I've update the PR so that flag set to true when:
- there are no such flag
- flag exist but value is not `true`

If the flag is already true and there are no there tags to update, ci won't make an additional call.